pkgmgmtutils/brew/formula_updater_test.go

```diff
@@ -0,0 +1,338 @@
+package brew_test
```

Comment:
we use this code to update formulas that aren't brew (e.g. fish-food), i'd recommend renaming the entire package

Reply:
including the file names